helm/charts/kratos/README.md

```diff
@@ -73,7 +73,7 @@ A ORY Kratos Helm chart for Kubernetes
 | kratos.config.serve.public.port | int | `4433` |  |
 | kratos.development | bool | `false` |  |
 | kratos.emailTemplates | object | `{}` | You can customize the emails kratos is sending (also uncomment config.courier.template_override_path below) |
-| kratos.identitySchemas | object | `{}` | You can add multiple identity schemas here |
+| kratos.identitySchemas | object | `{}` | You can add multiple identity schemas here. You can pass JSON schema using `--set-file` Helm CLI argument. |
```

Comment:
This is generated from comments in the values.yaml please move the description there :)